.github/workflows/publish.yml

```diff
@@ -28,52 +28,15 @@ jobs:
     name: Push charts to the web
     runs-on: ubuntu-latest
     steps:
-      - name: stunner checkout
-        uses: actions/checkout@v3
-        with:
-          path: stunner
-          ref: main
-          repository: l7mp/stunner
+
       - name: Get the version
         id: vars
         run: echo ::set-output name=tag::$(echo ${GITHUB_REF:11})
-      - name: l7mp.io checkout
-        uses: actions/checkout@v3
-        with:
-          token: ${{ secrets.WEB_PAT_TOKEN }}
-          path: l7mp.io
-          ref: master
-          repository: l7mp/l7mp.io
-      - name: stunner-helm checkout
-        uses: actions/checkout@v3
+
+      - name: Repository dispatch with passing the tagged version
+        uses: peter-evans/repository-dispatch@v2.1.1
```

Comment:
I recommend to fix the major version only to enable potential fixes in the later minor/patch releases.
```diff
+        uses: peter-evans/repository-dispatch@v2
-        uses: peter-evans/repository-dispatch@v2.1.1
```